### DISCUSSION TITLE
why the documentation is a mess

### DISCUSSION THREAD

**Discussion by nCold** [django-cms/django-cms#7323] [General]
Hello pyhoners and django lovers, so i wanted to try Django cms 
decided to try the manual way in the documentation that should be a straight forward copy past cmd

the documentation is written in a messy way as it keeps going back and forth back and forth, 
say for example you tackle what lines should be in  MIDDLEWARE  and what each line do, you should put only lines that needs to be copy-past ready, not go from what should be in template then to language then back to template then to installed_apps then middleware then installed apps....
why is written that way ??

**Comment by vinitkumar** [comment]
Hi @nCold Thank you for raising the issue. We would love to hear more about how the documentation was not helpful for you. Could you please also let us know your existing Python/Django experience. We want the documentation to be accessible to users with all levels of experience and would appreciate constructive suggestions in which we can make it better. 

Also, since this projects like many other Open Source projects is run via contributors, please be kind and respectful in the conversation and we would do our best to improve it.
